src/layouts/main.tsx

```diff
@@ -8,13 +8,17 @@ import { Header } from "src/components/common/Header";
 import { Footer } from "src/components/common/Footer";
 import { useExposeEnableYAFunction } from "src/common/useExposeEnableYAFunction";
 import config from "src/config";
+import "../i18n";
```

Comment:
This seems to be importing for side effects. Is there a way to avoid that, such as by changing all the top-level variables defined in that file to exports and importing them?

Reply:
Yeah, copied this from the docs but I agree it's weird, will update.